src/ConsoleBuffer/ConsoleWrapper.cs

```diff
@@ -183,7 +183,7 @@ private void ReadConsoleTask()
         {
             using (var ptyOutput = new FileStream(this.readHandle, FileAccess.Read))
             {
-                var input = new byte[2048];
+                var input = new byte[4096];
```

Comment:
Add a comment explaining why this size was chosen? Might also make it a const declared somewhere near the top of the class?

Reply:
Done